en/Getting started/Sync your notes across devices.md

```diff
@@ -0,0 +1,111 @@
+Syncing means keeping your notes up to date across your devices, such as your laptop and phone.
+
+The easiest way to sync your notes between your devices is to use [[Introduction to Obsidian Sync|Obsidian Sync]]. If you already have an Obsidian Sync subscription, see how to [[Set up Obsidian Sync]].
+
+If you don't want to use Obsidian Sync, this page lists alternative methods for syncing your vaults with multiple devices.
+
+## Sync notes across multiple desktops
+
+If you don't use Obsidian on your mobile devices, you can use any third-party service that syncs a local folder to a cloud storage.
+
+For example, you can use any of the following services:
+
+- Dropbox
+- Google Drive
+- iCloud Drive
+- OneDrive
+- Syncthing
+
+To sync your notes, follow the instructions for the service you're using to sync a folder on your local file system. Then open the folder as an existing vault on all your desktop devices.
+
+## Sync your notes on iPhone and iPad
+
+To sync your notes to your iPhone or iPad, we officially support the following options:
+
+- [[Introduction to Obsidian Sync|Obsidian Sync]]
+- [[#iCloud Drive]]
+
+**Note:** The following services are not supported. If you discover a way to sync your notes on your iOS device using any of these services, let us know on our community channels.
+
+- Dropbox
+- Google Drive
+- OneDrive
+- Syncthing
+
+### iCloud Drive
```

Comment:
I think the flow here is a little confusing; it's not very clear we're starting to talk about mobile workflows at this point -- did I miss something?

Reply:
Could you elaborate on this?

"iCloud Drive" is a subheading under "Sync your notes on iPhone and iPad", and it's listed as one of the two supported methods for syncing notes on iPhone and iPad?

Reply:
Oh nevermind, I misread, sorry about that!